app/api/auth.py

```diff
@@ -269,3 +273,22 @@ def change_password():
         "name": user.fullname if user.fullname else None,
         "password-changed": True
     })
+
+@ticket_blueprint.route('/tickets/attendees/tickets/pdf/<string:order_identifier>')
+@jwt_required()
+def ticket_attendee_authorized(order_identifier):
+    if current_user:
+        try:
+            order = Order.query.filter_by(identifier=order_identifier).first()
+            user_id = order.user.id
+        except NoResultFound:
+            return NotFoundError({'source': ''}, 'This ticket is not associated with any order').respond()
+        if current_user.id == user_id:
+            file_path = os.path.join('/tickets/attendees/tickets/pdf/', order_identifier + '/' + order_identifier + '.pdf')
```

Comment:
Shouldn't this be generated on the fly if not already present?

Reply:
Could you emphasize on this? Didn't really get how we could generate file paths on the go.

Reply:
I've imported from the UPLOAD_PATHS. Did you mean something else though?